internal/provider/provider.go

```diff
@@ -47,6 +48,11 @@ func (p *ProwlarrProvider) Schema(_ context.Context, _ provider.SchemaRequest, r
 				Optional:            true,
 				Sensitive:           true,
 			},
+			"authorization": schema.StringAttribute{
+				MarkdownDescription: "Token for token-based authentication with Prowlarr. This is an alternative to using an API key. Set this via the `PROWLARR_AUTHORIZATION` environment variable. Either `authorization` or `api_key` must be provided, but not both.",
```

Comment:
```suggestion
				MarkdownDescription: "Token for token-based authentication with Prowlarr. This is an alternative to using an API key. Set this via the `PROWLARR_AUTHORIZATION` environment variable. One of `authorization` or `api_key` must be provided, but not both.",
```

just a nit, and up to preference

Reply:
fixed.